src/js/control-bar/volume-menu-button.js

```diff
@@ -76,6 +82,19 @@ class VolumeMenuButton extends MenuButton {
   }
 
   /**
+   * Generates a class name with the appropriate orientation for the slider
+   *
+   * @method orientationClassName
+   */
+  orientationClassName() {
```

Comment:
Unless you're using this somewhere else you could probably just have this logic in the buildCSSClass function.